### ISSUE TITLE
TypeError: args[0].replace is not a function

### ISSUE THREAD

**Issue by boljoro** [jsoverson/hackium#39]
```
Error: Hackium failed during bootup and may be in an unstable state.
(node:14487) UnhandledPromiseRejectionWarning: TypeError: args[0].replace is not a function
    at Logger.format (/home/chris/node_modules/hackium/dist/src/util/logger.js:14:27)
    at Logger.error (/home/chris/node_modules/hackium/dist/src/util/logger.js:39:59)
    at /home/chris/node_modules/hackium/dist/src/cli.js:81:13
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

It should be a string to use the replace function, `args[0].toString().replace`.

**Comment by jsoverson** [comment]
Fixed in 1.0.8, thanks @boljoro

**Comment by jsoverson** [comment]
1.0.9, rather
